tutorials/shaders/compute_shaders.rst

```diff
@@ -206,7 +206,7 @@ So let's initialize an array of floats and create a storage buffer:
     var input_bytes := input.to_byte_array()
 
     # Create a storage buffer that can hold our float values.
-    # Each float has 8 byte (32 bit) so 10 x 8 = 80 bytes
+    # Each float has 4 byte (32 bit) so 10 x 4 = 40 bytes
```

Comment:
```suggestion
    # Each float has 4 bytes (32 bit) so 10 x 4 = 40 bytes
```
I know this issue was present before you change. But we should fix the whole line at the same time